SPECS/ruby/ruby.spec

```diff
@@ -190,7 +286,57 @@ sudo -u test make test TESTS="-v"
 %dir %{rubygems_dir}
 %{rubygems_dir}/rubygems
 
+%files -n rubygem-bigdecimal
+%{_libdir}/gems/%{name}/bigdecimal-%{bigdecimal_version}
+%{gem_dir}/gems/bigdecimal-%{bigdecimal_version}
+
+%files -n rubygem-minitest
+%dir %{gem_dir}/gems/minitest-%{minitest_version}
+%exclude %{gem_dir}/gems/minitest-%{minitest_version}/.*
+%{gem_dir}/gems/minitest-%{minitest_version}/Manifest.txt
+%{gem_dir}/gems/minitest-%{minitest_version}/design_rationale.rb
+%{gem_dir}/gems/minitest-%{minitest_version}/lib
+%{gem_dir}/specifications/minitest-%{minitest_version}.gemspec
+%doc %{gem_dir}/gems/minitest-%{minitest_version}/History.rdoc
+%doc %{gem_dir}/gems/minitest-%{minitest_version}/README.rdoc
+%{gem_dir}/gems/minitest-%{minitest_version}/Rakefile
+%{gem_dir}/gems/minitest-%{minitest_version}/test
+
+%files -n rubygem-power_assert
+%dir %{gem_dir}/gems/power_assert-%{power_assert_version}
+%exclude %{gem_dir}/gems/power_assert-%{power_assert_version}/.*
+%license %{gem_dir}/gems/power_assert-%{power_assert_version}/BSDL
+%license %{gem_dir}/gems/power_assert-%{power_assert_version}/COPYING
+%license %{gem_dir}/gems/power_assert-%{power_assert_version}/LEGAL
+%{gem_dir}/gems/power_assert-%{power_assert_version}/lib
+%{gem_dir}/specifications/power_assert-%{power_assert_version}.gemspec
+%{gem_dir}/gems/power_assert-%{power_assert_version}/Gemfile
+%{gem_dir}/gems/power_assert-%{power_assert_version}/Rakefile
+
+%files -n rubygem-rdoc
+%{_bindir}/rdoc
+%{_bindir}/ri
+%{gem_dir}/gems/rdoc-%{rdoc_version}
+%{_mandir}/man1/ri*
+
+%files -n rubygem-test-unit
+%dir %{gem_dir}/gems/test-unit-%{test_unit_version}
+%license %{gem_dir}/gems/test-unit-%{test_unit_version}/GPL
+%license %{gem_dir}/gems/test-unit-%{test_unit_version}/LGPL
+%license %{gem_dir}/gems/test-unit-%{test_unit_version}/COPYING
+%license %{gem_dir}/gems/test-unit-%{test_unit_version}/PSFL
+%{gem_dir}/gems/test-unit-%{test_unit_version}/lib
+%{gem_dir}/gems/test-unit-%{test_unit_version}/sample
+%{gem_dir}/gems/test-unit-%{test_unit_version}/test
+%{gem_dir}/specifications/test-unit-%{test_unit_version}.gemspec
+%doc %{gem_dir}/gems/test-unit-%{test_unit_version}/README.md
+%{gem_dir}/gems/test-unit-%{test_unit_version}/Rakefile
+%doc %{gem_dir}/gems/test-unit-%{test_unit_version}/doc
+
 %changelog
+* Tue Mar 15 2022 Neha Agarwal <nehaagarwal@microsoft.com> - 2.7.4-2
+- Build bigdecimal, minitest, test-unit, rdoc and power_assert rubygems (taken from Fedora)
```

Comment:
Please also mention the version of Fedora and the license under which they release their spec (I'm guessing it's MIT).